vendor/sinon.js

```diff
@@ -9864,7 +9864,7 @@ process.umask = function() { return 0; };
         // Unlike global isNaN, this avoids type coercion
         // typeof check avoids IE host object issues, hat tip to
         // lodash
-        var val = value; // JsLint thinks value !== value is "weird"
```

Comment:
No need to touch the vendored sinon.  It's more-or-less copied straight (the upstream package pulls in an unnecessary dep, so we don't use it).

Reply:
Undone!